app/models/user_session.rb

```diff
@@ -1,6 +1,15 @@
 class UserSession < ApplicationRecord
 
-  TTL = 2.weeks.freeze
+  TTL = case Rails.configuration.three_scale.user_sesion_ttl
```

Comment:
I also don't find the constant choice very flexible. This is the least disruptive way to implement the feature and it can change whenever we decide to allow configurable sessions per tenant and/or portal. Will keep constant if you don't mind.

will change to only seconds